{"uid":"43954c03f49cca12","name":"test_invalid_sendmoney[0.00001-tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24-Fee exceeds amount0]","fullName":"tests.wallet_tests.test_wallet_send_money#test_invalid_sendmoney","historyId":"e108058fe6d1adb7fbb51213604312eb","time":{"start":1734937516123,"stop":1734937518645,"duration":2522},"status":"failed","statusMessage":"Failed: Error during message validation: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nassert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\n  \n  - Fee exceeds amount\n  + Minimum amount is 0.00001 BEL","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e4869007641110b4803f74b0cf913651\")>\namount = '0.00001', blank = 'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'\nexpected_error = 'Fee exceeds amount'\n\n    @pytest.mark.usefixtures(\"driver\")\n    @allure.feature(\"Sending money with an invalid balance\")\n    @pytest.mark.parametrize(\"amount, blank, expected_error\", [\n        (\"555555555\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"There's not enough money in your account\"),\n        (\"\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Minimum amount is 0.00001 BEL\"),\n        (\"0.1\", \"\", \"Invalid receiver's address\"),\n        (\"\", \"\", \"Invalid receiver's address\"),\n        (\"\", f\"{Data.NOT_VALID_ADDRESS}\", \"Invalid receiver's address\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\")\n        ])\n    \n    def test_invalid_sendmoney(driver, amount, blank, expected_error):\n    \n        ex_id, password = restore_by_private_key_proc(driver)\n    \n        change_network = ManePage(driver)\n    \n        change_network.change_network(ex_id)\n        change_network.get_balance()\n        change_network.send_page_btn()\n    \n        send_invalid_amount = SendPage(driver)\n    \n        send_invalid_amount.enter_address(blank)\n        send_invalid_amount.enter_amount(amount)\n        send_invalid_amount.include_fee()\n        send_invalid_amount.cont_send_money()\n    \n        try:\n            error_message = WebDriverWait(driver, 5).until(\n                EC.element_to_be_clickable((By.CLASS_NAME, \"error\"))\n            )\n            assert error_message.is_displayed(), \"The error message is not displayed\"\n>           assert error_message.text == expected_error, f\"An error message was expected: '{expected_error}', but received: '{error_message.text}'\"\nE           AssertionError: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nE           assert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\nE             \nE             - Fee exceeds amount\nE             + Minimum amount is 0.00001 BEL\n\nwallet_tests/test_wallet_send_money.py:91: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e4869007641110b4803f74b0cf913651\")>\namount = '0.00001', blank = 'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'\nexpected_error = 'Fee exceeds amount'\n\n    @pytest.mark.usefixtures(\"driver\")\n    @allure.feature(\"Sending money with an invalid balance\")\n    @pytest.mark.parametrize(\"amount, blank, expected_error\", [\n        (\"555555555\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"There's not enough money in your account\"),\n        (\"\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Minimum amount is 0.00001 BEL\"),\n        (\"0.1\", \"\", \"Invalid receiver's address\"),\n        (\"\", \"\", \"Invalid receiver's address\"),\n        (\"\", f\"{Data.NOT_VALID_ADDRESS}\", \"Invalid receiver's address\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\")\n        ])\n    \n    def test_invalid_sendmoney(driver, amount, blank, expected_error):\n    \n        ex_id, password = restore_by_private_key_proc(driver)\n    \n        change_network = ManePage(driver)\n    \n        change_network.change_network(ex_id)\n        change_network.get_balance()\n        change_network.send_page_btn()\n    \n        send_invalid_amount = SendPage(driver)\n    \n        send_invalid_amount.enter_address(blank)\n        send_invalid_amount.enter_amount(amount)\n        send_invalid_amount.include_fee()\n        send_invalid_amount.cont_send_money()\n    \n        try:\n            error_message = WebDriverWait(driver, 5).until(\n                EC.element_to_be_clickable((By.CLASS_NAME, \"error\"))\n            )\n            assert error_message.is_displayed(), \"The error message is not displayed\"\n            assert error_message.text == expected_error, f\"An error message was expected: '{expected_error}', but received: '{error_message.text}'\"\n        except Exception as e:\n>           pytest.fail(f\"Error during message validation: {e}\")\nE           Failed: Error during message validation: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nE           assert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\nE             \nE             - Fee exceeds amount\nE             + Minimum amount is 0.00001 BEL\n\nwallet_tests/test_wallet_send_money.py:93: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1734937387619,"stop":1734937387620,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1734937515538,"stop":1734937516121,"duration":583},"status":"passed","steps":[],"attachments":[{"uid":"7a80f988911dbea2","name":"Driver","source":"7a80f988911dbea2.txt","type":"text/plain","size":20}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"testStage":{"status":"failed","statusMessage":"Failed: Error during message validation: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nassert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\n  \n  - Fee exceeds amount\n  + Minimum amount is 0.00001 BEL","statusTrace":"driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e4869007641110b4803f74b0cf913651\")>\namount = '0.00001', blank = 'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'\nexpected_error = 'Fee exceeds amount'\n\n    @pytest.mark.usefixtures(\"driver\")\n    @allure.feature(\"Sending money with an invalid balance\")\n    @pytest.mark.parametrize(\"amount, blank, expected_error\", [\n        (\"555555555\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"There's not enough money in your account\"),\n        (\"\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Minimum amount is 0.00001 BEL\"),\n        (\"0.1\", \"\", \"Invalid receiver's address\"),\n        (\"\", \"\", \"Invalid receiver's address\"),\n        (\"\", f\"{Data.NOT_VALID_ADDRESS}\", \"Invalid receiver's address\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\")\n        ])\n    \n    def test_invalid_sendmoney(driver, amount, blank, expected_error):\n    \n        ex_id, password = restore_by_private_key_proc(driver)\n    \n        change_network = ManePage(driver)\n    \n        change_network.change_network(ex_id)\n        change_network.get_balance()\n        change_network.send_page_btn()\n    \n        send_invalid_amount = SendPage(driver)\n    \n        send_invalid_amount.enter_address(blank)\n        send_invalid_amount.enter_amount(amount)\n        send_invalid_amount.include_fee()\n        send_invalid_amount.cont_send_money()\n    \n        try:\n            error_message = WebDriverWait(driver, 5).until(\n                EC.element_to_be_clickable((By.CLASS_NAME, \"error\"))\n            )\n            assert error_message.is_displayed(), \"The error message is not displayed\"\n>           assert error_message.text == expected_error, f\"An error message was expected: '{expected_error}', but received: '{error_message.text}'\"\nE           AssertionError: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nE           assert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\nE             \nE             - Fee exceeds amount\nE             + Minimum amount is 0.00001 BEL\n\nwallet_tests/test_wallet_send_money.py:91: AssertionError\n\nDuring handling of the above exception, another exception occurred:\n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"e4869007641110b4803f74b0cf913651\")>\namount = '0.00001', blank = 'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'\nexpected_error = 'Fee exceeds amount'\n\n    @pytest.mark.usefixtures(\"driver\")\n    @allure.feature(\"Sending money with an invalid balance\")\n    @pytest.mark.parametrize(\"amount, blank, expected_error\", [\n        (\"555555555\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"There's not enough money in your account\"),\n        (\"\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Minimum amount is 0.00001 BEL\"),\n        (\"0.1\", \"\", \"Invalid receiver's address\"),\n        (\"\", \"\", \"Invalid receiver's address\"),\n        (\"\", f\"{Data.NOT_VALID_ADDRESS}\", \"Invalid receiver's address\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\"),\n        (\"0.00001\", f\"{Data.VALID_ADDRESS_FOR_CHECK}\", \"Fee exceeds amount\")\n        ])\n    \n    def test_invalid_sendmoney(driver, amount, blank, expected_error):\n    \n        ex_id, password = restore_by_private_key_proc(driver)\n    \n        change_network = ManePage(driver)\n    \n        change_network.change_network(ex_id)\n        change_network.get_balance()\n        change_network.send_page_btn()\n    \n        send_invalid_amount = SendPage(driver)\n    \n        send_invalid_amount.enter_address(blank)\n        send_invalid_amount.enter_amount(amount)\n        send_invalid_amount.include_fee()\n        send_invalid_amount.cont_send_money()\n    \n        try:\n            error_message = WebDriverWait(driver, 5).until(\n                EC.element_to_be_clickable((By.CLASS_NAME, \"error\"))\n            )\n            assert error_message.is_displayed(), \"The error message is not displayed\"\n            assert error_message.text == expected_error, f\"An error message was expected: '{expected_error}', but received: '{error_message.text}'\"\n        except Exception as e:\n>           pytest.fail(f\"Error during message validation: {e}\")\nE           Failed: Error during message validation: An error message was expected: 'Fee exceeds amount', but received: 'Minimum amount is 0.00001 BEL'\nE           assert 'Minimum amou...s 0.00001 BEL' == 'Fee exceeds amount'\nE             \nE             - Fee exceeds amount\nE             + Minimum amount is 0.00001 BEL\n\nwallet_tests/test_wallet_send_money.py:93: Failed","steps":[],"attachments":[{"uid":"491473300012fd6f","name":"Current ID","source":"491473300012fd6f.txt","type":"text/plain","size":44},{"uid":"b72136b73ae26b6f","name":"Password","source":"b72136b73ae26b6f.txt","type":"text/plain","size":73},{"uid":"b9ebf99bd0deae9b","name":"Action","source":"b9ebf99bd0deae9b.txt","type":"text/plain","size":20},{"uid":"fb9fc5f463ceffc3","name":"Action","source":"fb9fc5f463ceffc3.txt","type":"text/plain","size":40},{"uid":"9cee51b72f8c7727","name":"Action","source":"9cee51b72f8c7727.txt","type":"text/plain","size":34},{"uid":"1bb37d48f76aa1ae","name":"Action","source":"1bb37d48f76aa1ae.txt","type":"text/plain","size":23},{"uid":"f73d30d36e540584","name":"Action","source":"f73d30d36e540584.txt","type":"text/plain","size":38},{"uid":"fa669e72406e243f","name":"Action","source":"fa669e72406e243f.txt","type":"text/plain","size":21},{"uid":"cbd5944c5ee058ce","name":"Navigation","source":"cbd5944c5ee058ce.txt","type":"text/plain","size":46},{"uid":"ed6cb0474d03378f","name":"Action","source":"ed6cb0474d03378f.txt","type":"text/plain","size":32},{"uid":"b686dde3baf3f098","name":"Balance","source":"b686dde3baf3f098.txt","type":"text/plain","size":34},{"uid":"9cd393e40ae5a83d","name":"Action","source":"9cd393e40ae5a83d.txt","type":"text/plain","size":22},{"uid":"4c167b3dcaf367e0","name":"Action","source":"4c167b3dcaf367e0.txt","type":"text/plain","size":23},{"uid":"36f7a4418aff387f","name":"Action","source":"36f7a4418aff387f.txt","type":"text/plain","size":38},{"uid":"de4dfedde71673a1","name":"Action","source":"de4dfedde71673a1.txt","type":"text/plain","size":39},{"uid":"ebce05aa7ed52e27","name":"Action","source":"ebce05aa7ed52e27.txt","type":"text/plain","size":22}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":16,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1734937518650,"stop":1734937518803,"duration":153},"status":"passed","steps":[],"attachments":[{"uid":"1899b56e8ec2f7f4","name":"Screenshot","source":"1899b56e8ec2f7f4.png","type":"image/png","size":36564}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"feature","value":"Sending money with an invalid balance"},{"name":"tag","value":"@pytest.mark.usefixtures('driver')"},{"name":"parentSuite","value":"tests.wallet_tests"},{"name":"suite","value":"test_wallet_send_money"},{"name":"host","value":"01bad6ed0c6f"},{"name":"thread","value":"8-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.wallet_tests.test_wallet_send_money"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"amount","value":"'0.00001'"},{"name":"blank","value":"'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'"},{"name":"expected_error","value":"'Fee exceeds amount'"}],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["@pytest.mark.usefixtures('driver')"]},"source":"43954c03f49cca12.json","parameterValues":["'0.00001'","'tbel1q0v7cjajnlkll0ppjsmdduwh2gfjml3nhmuks24'","'Fee exceeds amount'"]}